.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @loadsmart/<CODE-OWNER>
```

Comment:
@raphapr is there a catch-all team that we could assign this one to?

Reply:
@barbosa yes, there is `@loadsmart/all-teams`, I just updated the owner